src/main/java/org/folio/inventory/consortium/services/EntitiesLinksService.java

```diff
@@ -0,0 +1,14 @@
+package org.folio.inventory.consortium.services;
+
+import io.vertx.core.Future;
+import org.folio.Link;
+import org.folio.LinkingRuleDto;
+import org.folio.inventory.common.Context;
+
+import java.util.List;
+
+public interface EntitiesLinksService {
```

Comment:
nit: I don't think this interface is consortium specific. Could be put in a package outside of "consortium"

Reply:
Moved from consortium package, thanks